.tests/testScript_plotting.m

```diff
@@ -161,7 +161,7 @@ zlabel sqrt(rpm)
 % make sure plot doesn't look at wrong scd and instead gets from displayunits
 plot3(scd(c,'hp'),scd(a*u.m,'K'),scd(c.*a,'lb_m'),'LineStyle',':') 
 xlabel mass; ylabel length; zlabel mass
-assert(axishasunits('lb','ft','lb_m'))
+assert(axishasunits('kg','m','lb_m'))
```

Comment:
See previous comment. The preferred display units for this test drive it to lb and ft.

Reply:
solved with https://github.com/iAnyKey/physical-units-for-matlab/commit/3bb16f89dc39a85a5f0e0396d7056e85042b4035